backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -28,6 +28,29 @@ include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
 include::modules/migration-debugging-velero-resources.adoc[leveloffset=+1]
 
+[id="oadp-pod-crash-resource-request"]
+== Pods fail or restart due to lack of memory or CPU
+
+If a `Velero` pod or a `Restic` pod fails or restarts due to a lack of memory or CPU, you can set specific resource requests for either of those resources.
```

Comment:
@RichardHoch, I don't think we need to use back ticks for these, which appear to be program/application names. We're using back ticks for object names, like

> `NetworkPolicy` object